charts/project-origin-wallet/templates/migrate-job.yaml

```diff
@@ -17,23 +17,29 @@ spec:
           args:
             - "--migrate"
           env:
-          {{ if .Values.persistance.cloudNativePG.enabled }}
+          {{- if eq .Values.persistence.type "CloudNativePG" }}
             - name: DB_HOST
-              value: {{ .Values.persistance.cloudNativePG.name }}-rw
+              value: {{ .Values.persistence.cloudNativePG.name }}-rw
             - name: DB_PORT
               value: "5432"
             - name: DB_DATABASE
-              value: {{ .Values.persistance.cloudNativePG.database }}
+              value: {{ .Values.persistence.cloudNativePG.database }}
             - name: DB_USERNAME
               valueFrom:
                 secretKeyRef:
-                  name: {{ .Values.persistance.cloudNativePG.name }}-app
+                  name: {{ .Values.persistence.cloudNativePG.name }}-app
                   key: username
             - name: DB_PASSWORD
               valueFrom:
                 secretKeyRef:
-                  name: {{ .Values.persistance.cloudNativePG.name }}-app
+                  name: {{ .Values.persistence.cloudNativePG.name }}-app
                   key: password
             - name: ConnectionStrings__Database
               value: Host=$(DB_HOST); Port=$(DB_PORT); Database=$(DB_DATABASE); Username=$(DB_USERNAME); Password=$(DB_PASSWORD);
+          {{- else if eq .Values.persistence.type "BYOD" }}
+            - name: ConnectionStrings__Database
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.persistence.byod.secretName }}
+                  key: connectionString
```

Comment:
should use the secretKey

Reply:
Done:

https://github.com/project-origin/wallet/pull/254/commits/35008b6124f684cecba21982ec9dd06093d28977